README.md

```diff
@@ -6,6 +6,22 @@ This experiment creates and trains three neural networks, termed Alice, Bob, and
 
 This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/)
 
+## Model Metadata
+| Domain | Application | Industry  | Framework | Training Data | Input Data Format |
+| ------------- | --------  | -------- | --------- | --------- | -------------- |
+| Cryptography | Adversarial Neural Cryptography | Security | TensorFlow | Auto-generated | N/A |
```

Comment:
Maybe you should use the metadata Nick created for this model?
https://github.ibm.com/IBMCode/Code-Models/blob/master/max-adversarial-cryptography/index.md

Reply:
I'll use this, thanks